docs/ui/insight/get-started.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Get started with RedisInsight
+linkTitle: Get started
+description: Get started with RedisInsight
+weight: 40
+
+---
+
+RedisInsight is available as a standalone download or [bundled with Redis Stack](/docs/stack/get-started/install). Install it and try the in-app examples available in Workbench.
+
+
+You can use RedisInsight on MacOS, Linux, or Windows.
+
+## Install on MacOS
```

Comment:
looks great
one comment - let's firstly describe Windows, then Mac, then Linux (based on the number of users on these platforms)